### PR TITLE
What about the ChapGPT?

### PR DESCRIPTION
**Pull request by wpschaub**
A question that made me speechless at a recent Navigating the Future event.

### CODE REVIEW

File: content/what-about-chatgpt.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ 
+Respect as I programmed on the Intel 80186 processor, which had a 16-bit data width capable of addressing a phenomenal 65,536 bytes of memory. The magnitude of processing, memory space, and artificial intelligence we enjoy today makes engineers like a tad antiquated. 
+
+![Old](../images/what-about-chatgpt-3.png)
+ 
+> “I'm old but I'm not obsolete." Terminator, aka Arnold Schwarzenegger
+
+---
+
+# I loathe the dump chat bots
```
Comment (MerlinDarkstar) [inline content/what-about-chatgpt.md]: dumb instead of dump :)

File: content/what-about-chatgpt.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+# Having a conversation with an intelligent bot sound appealing
+
+![AI Conversation](../images/what-about-chatgpt-5.png)
+ 
+What excites me about artificial intelligence is that we can turn the dialog with the machine into a meaningful conversation, making it possible for the machine to continuously learn, ask clarification questions, and investigate and formulate a detailed response in the background. Multi, continuous, repetitive, and context switching tasking is no problem for our binary companions.
+
+I dream of a common engineering bot that can be the friendly onboarding buddy, an engineer’s patient wingman, and the first point contact for all technical and operational support. If anyone wants to raise this as an idea for the upcoming InnoFest / Hackathon, then you have both by vote and support!
```
Comment (MerlinDarkstar) [inline content/what-about-chatgpt.md]: both by vote and support! 
to
both my vote and support!

Comment (wpschaub) [reply content/what-about-chatgpt.md]: Fixed

File: content/what-about-chatgpt.md (hunk 1 above)
Comment (wpschaub) [reply content/what-about-chatgpt.md]: Fixed

**Review by MerlinDarkstar — APPROVED** [review]
Nice - love the final words by your son, nice wrap-up.

**Review by AndreKaminski — APPROVED** [review]

**Pull request merged**